ArduSub/ArduSub.cpp

```diff
@@ -183,7 +183,8 @@ void Sub::ten_hz_logging_loop()
     // log attitude data if we're not already logging at the higher rate
     if (should_log(MASK_LOG_ATTITUDE_MED) && !should_log(MASK_LOG_ATTITUDE_FAST)) {
         Log_Write_Attitude();
-        ahrs_view.Write_Rate(motors, attitude_control, pos_control);
+        attitude_control.Write_Attitude();
+        attitude_control.Write_Rate(pos_control);
```

Comment:
Write_attitude already logs multiple messages, perhaps we should have a generic, "attitude control, do your own logging now" call.

I'd actually prefer the attitude controller to have its own logging control parameters, but this might be divisive.

Reply:
Actually no - they will be called from different threads at different rates so need to be kept different. Sub logging is already messed up - really someone needs to go in and decide whether they really need all this stuff at high rates.